### PR TITLE
Create review

### PR DESCRIPTION
**Pull request by kiverso**
Creates a form for users to fill out to post a review for a shelter. Reviews are created and stored in the database after the user fill out the required fields and hits submit.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -15,4 +15,6 @@
   get "pets/:id/edit", to: 'pets#edit'
   patch 'pets/:id', to: 'pets#update'
   delete 'pets/:id', to: 'pets#destroy'
+  get '/shelters/:shelter_id/reviews/new', to: 'reviews#new'
+  post '/shelters/:shelter_id/reviews', to: 'reviews#create'
```
Comment (kiverso) [inline config/routes.rb:19]: Let me know what you think of this routing for creating a review. Seems right but also felt a little bit off given that there is not reviews page for a shelter, it is right on the show page.

Comment (Rostammahabadi) [reply config/routes.rb:19]: That's exactly how i have the edit so this is perfect

File: app/views/reviews/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+</div>
+
+<div class="input-form">
+    <%= form_tag "/shelters/#{@shelter_id}/reviews", method: :post do %>
+        <div class="row form-entry">
+            <%= label_tag :title, nil, class: "col-2 light-text" %>
+            <%= text_field_tag :title, nil, class: "col-5", required: true %>
+
+            <%= label_tag :rating, nil, class: "col-3 light-text" %>
+            <%= select_tag :rating, options_for_select([1, 2, 3, 4, 5]), required: true %>
```
Comment (Rostammahabadi) [inline app/views/reviews/new.html.erb:12]: Is this a drop down menu for selection? (1-5)

Comment (kiverso) [reply app/views/reviews/new.html.erb:12]: Yeah, it just gives a drop down so a user can't submit anything other than a 1-5 rating.

Comment (Rostammahabadi) [reply app/views/reviews/new.html.erb:12]: perfect, this looks great!

**Pull request merged**